CHANGELOG.md

```diff
@@ -3,11 +3,13 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 # [v1.3.0](unreleased)
 ### Notes
-- This release supports for API 800,1000,1200  for resource listed below.
+- This release supports for API 800,1000,1200,1600  for resource listed below.
 
 ### Oneview Features supported
 - Hypervisor Manager
 - Server Certificate
+- Storage System
```

Comment:
Is the Storage system supported in this PR? If so, please include it in the PR name as well, and if not please remove this entry. Hypervisor manager and server certificate resources are not yet supported for API 1600 version, remove these entries as well.

Reply:
I have included both storage systema and volume attachment in this PR itself as the changes were minimal